static/app/components/events/autofix/autofixSteps.tsx

```diff
@@ -280,6 +281,7 @@ export function AutofixSteps({data, groupId, runId}: AutofixStepsProps) {
             </div>
           );
         })}
+        <AutofixOutputStream step={lastStep} />
```

Comment:
see other comment, don't even need to render this if the step doesn't have a stream